Sprint-3/3-mandatory-practice/implement/repeat.js

```diff
@@ -1,5 +1,7 @@
-function repeat() {
-    return "hellohellohello";
+function repeat(str, count) {
+    if (count < 0) throw new Error("Error");
+    return str.repeat(count);
+  
```

Comment:
You're not decrementing count so this function will recurse infinitely. THere needs to be a stopping condition that is reached.